app/models/va_facility.rb

```diff
@@ -1,11 +1,12 @@
 # frozen_string_literal: true
-require 'facilities/async_client'
-require 'facilities/multi_client'
+require 'facilities/local_store'
 
 class VAFacility < ActiveModelSerializers::Model
   attr_accessor :unique_id, :name, :facility_type, :classification, :website,
                 :lat, :long, :address, :phone, :hours, :services, :feedback
 
+  @lock = Mutex.new
```

Comment:
I can't figure out why this mutex would be necessary. It's not standard in Rails apps to put a mutex around the setting of class variables and I don't think there's anything special here that would require it. But I might be totally confused.